apps/charterafrica/src/components/FeaturedPostCard/FeaturedPostCard.js

```diff
@@ -0,0 +1,117 @@
+import { Section } from "@commons-ui/core";
+import { Link } from "@commons-ui/next";
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  Box,
+  CardActions,
+} from "@mui/material";
+import React from "react";
+
+import { neutral, secondary } from "@/charterafrica/colors";
+import LineClampedRichTypography from "@/charterafrica/components/LineClampedRichTypography";
+
+const FeaturedPostCard = React.forwardRef(function FeaturedPostCard(
+  props,
+  ref
+) {
+  const { article } = props;
+
+  if (!article) return null;
+
+  const { title, date, excerpt, image, sx, link, author } = article;
+
```

Comment:
```suggestion
  const { title, date, excerpt, image, sx, link, author } = props;
  
  if (!title?.length) {
    return null;
  }
```

No need to pass `article` as an object.